.github/workflows/test-docker.yaml

```diff
@@ -1,8 +1,6 @@
 name: Test Docker image
 on:
   push:
-  pull_request:
-    types: [opened, reopened]
```

Comment:
could you put this back in?

Thanks

Reply:
actually ill do this

Reply:
Yeah wasn't sure if we also wanted that here so took it out